diff --git a/dist/esm/useNavigate.js b/dist/esm/useNavigate.js
index d7815852d1030da81320d3de47965283ae7ea77e..7580d213fa13de63bce4e2d5f9d3b0f1cc2f2a08 100644
--- a/dist/esm/useNavigate.js
+++ b/dist/esm/useNavigate.js
@@ -10,13 +10,10 @@ function useNavigate(_defaultOpts) {
   return React.useCallback(
     (options) => {
       const from = options.from ?? (_defaultOpts == null ? void 0 : _defaultOpts.from) ?? state.matches[matchIndex].fullPath;
-      return navigate({
-        ...options,
-        from
-      });
+      return navigate(options);
     },
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    [_defaultOpts == null ? void 0 : _defaultOpts.from, navigate]
+    [navigate]
   );
 }
 function Navigate(props) {
