diff --git a/_lib/app/hooks/useConfig.js b/_lib/app/hooks/useConfig.js
index ca91ce06eb0cbf19761ad10c5615869f62686552..0f0c104f1d84584200ad4353de7bd299bbb3c060 100644
--- a/_lib/app/hooks/useConfig.js
+++ b/_lib/app/hooks/useConfig.js
@@ -4,6 +4,8 @@ import { sha256 } from '@noble/hashes/sha256';
 import { bytesToHex } from '@noble/hashes/utils';
 import { createContext, useContext, useEffect, useState } from 'react';
 import { deserializeConfig, serializeConfig } from '../../config.js';
+import { TopNavItem } from '~/components/TopNavItem.js'
+
 const ConfigContext = createContext(virtualConfig);
 export const configHash = import.meta.env.DEV
     ? bytesToHex(sha256(serializeConfig(virtualConfig))).slice(0, 8)
@@ -30,7 +32,13 @@ export function ConfigProvider({ children, config: initialConfig, }) {
         if (typeof window !== 'undefined' && import.meta.env.DEV)
             window.localStorage.setItem(`vocs.config.${configHash}`, serializeConfig(config));
     }, [config]);
-    return _jsx(ConfigContext.Provider, { value: config, children: children });
+    return _jsx(ConfigContext.Provider, { value: {
+        ...config,
+        topNav: [
+            ...config.topNav,
+            { element: _jsx(TopNavItem, {}) }
+        ]
+    }, children: children });
 }
 export function useConfig() {
     return useContext(ConfigContext);
